src/visualization/recipes_makie.jl

```diff
@@ -102,10 +104,62 @@ function iplot(u, mesh::UnstructuredMesh2D, equations, solver, cache;
   fig
 end
 
+function iplot(u, mesh, equations, solver, cache;
```

Comment:
For consistency, this and all other `iplot`/`iplot!` methods should accept the `plot_mesh` and `show_axis` keywords, shouldn't they? Otherwise it can be confusing for the user if some variants support a different API than others.

Reply:
The keywords weren't needed previously because there was one button for toggling the mesh. However, we don't have those buttons for this PR because a user may end up plotting an arbitrary number of fields and we'd need one button for each field. The keywords essentially replace the button for toggling mesh plotting. 

I think I can add the `plot_mesh` and `show_axis` keywords to the previous version of `iplot` and have them toggle whether or not this button is shown.

Reply:
I think this would be nice, yes :+1: By default it gives the user everything they want, but in case they use the same keywords as in the other `iplot` versions, they get expected behavior and not an error.

Reply:
This change is still missing, isn't it?

Reply:
Sort of. The keyword arg now does something, but I decided against removing the button entirely. Instead it just starts the plot without visualizing the mesh.

Reply:
But having a non-functional button - wouldn't that be even more confusing? Maybe I just do not understand exactly what this effects...

Reply:
Oh! No, the button still works. All that changed is that the button starts in the "off" position if `plot_mesh=false`.